.codeclimate.yml

```diff
@@ -1,28 +1,49 @@
 ---
-engines:
-  eslint:
+version: "2"
```

Comment:
Please also do a PR for the docs for how we configure the code climate parameters with refs to docs or explanations to the options